src/cli/shared_opts.rs

```diff
@@ -33,11 +33,21 @@ pub struct UserOutputOpts {
         value_name = "FORMAT",
         global = true
     )]
-    pub output_format: OutputFormat,
+    output_format: OutputFormat,
 
     /// Disable user output
     #[arg(long, global = true)]
-    pub no_user_output: bool,
+    no_user_output: bool,
```

Comment:
Removed `pub` from them to force usage of the new `effective_output_format()` method to prevent anyone else from making the same error I did when directly using `output_format`, see https://github.com/foresterre/cargo-msrv/pull/963/files#r1708224557.